email_template_qweb/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+========================
+QWeb for email templates
+========================
+
+This module was written to allow you to write email templates in QWeb instead
+of jinja2. The advantage here is that with QWeb, you can make use of
+inheritance and the ``call`` statement, which allows you to reuse designs and
+snippets in multiple templates, making your development process simpler. 
+Furthermore, QWeb views are easier to edit with the integrated ACE editor.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Select `QWeb` in the field `Body templating engine`
+* Select a QWeb view to be used to render the body field
+* Apart from QWeb's standard variables, you also have access to ``object`` and
+  ``email_template``, which are browse records of the current object and the
+  email template in use, respectively.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/205/10.0
+
+Demo data contains an example on how to separate corporate identity from a
+template's content.
+
+For further information, please visit:
```

Comment:
update whole README according to latest template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst